### PR TITLE
Upgrade blade ink 0.7.3

### LINKED ISSUE #24: Upgrade to ink 0.9.0
- Upgrade to latest ink version (0.9.0) by upgrading to latest blade ink (0.7.0)

### PR DESCRIPTION
**Pull request by wilmol**
Will close #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://travis-ci.com/wilmol/mutters.svg?branch=master)](https://travis-ci.com/wilmol/mutters)
```
Comment (rabidgremlin) [inline README.md]: This isn't going to work. Builds from your repo :)

File: buildpublish.gradle [hunk 2]
```diff
@@ -106,4 +105,13 @@ tasks.withType(FindBugs) {
         xml.enabled = false
         html.enabled = true
     }
-}
\ No newline at end of file
+}
+
+apply plugin: 'jacoco'
```
Comment (rabidgremlin) [inline buildpublish.gradle]: Should probably be in a separate PR (adding build + test coverage). But not too worry. I'll roll all of these into a new branch anyway.

**Comment by rabidgremlin** [comment]
Have changed the base branch to v7...

File: buildpublish.gradle (hunk 2 above)
Comment (wilmol) [reply buildpublish.gradle]: I merged into my master... weird how it got put here

**Comment by wilmol** [comment]
@rabidgremlin resolved... travis/codecov should work automatically cause its a public repo, I think?

**Comment by rabidgremlin** [comment]
> @rabidgremlin resolved... travis/codecov should work automatically cause its a public repo, I think?

I have set up travis/codecov for the main repo...

**Pull request merged**